app/models/passenger.rb

```diff
@@ -0,0 +1,14 @@
+class Passenger < ApplicationRecord
+  has_many :trips
+  validates :name, presence: true
+  validates :phone_num, presence: true
+
```

Comment:
I suggest you consider making a `request_trip` business logic method in the Passenger model here, then you could call it from the controller, and dry up that method